test/jdk/java/util/stream/GathererTest.java

```diff
@@ -349,7 +349,7 @@ public void testCompositionOfStatefulGatherers(Config config) {
     @ParameterizedTest
     @MethodSource("configurations")
     public void testMassivelyComposedGatherers(Config config) {
-        final int ITERATIONS = 512; // Total number of compositions is 1 + (iterations*2)
+        final int ITERATIONS = 128; // Total number of compositions is 1 + (iterations*2)
```

Comment:
Sorry, but I think it is not appropriate to do a change like this here. It should have its own PR and discussion.

Reply:
Yes, I will revert this change and let @viktorklang-ora fine-tune it in another PR.

Reply:
Viktor has the change in JDK-8321124 to dial it down. Right now it requires too much stack on x86_32 systems used in the GHA testing.